### ISSUE TITLE
le whoami and le ls do not work without a host/agent key

### ISSUE THREAD

**Issue by mrud** [logentries/le#112]
`le whoami` and `le ls logs` do not work with just the user-key set.

Better error handling and documentation regarding these two commands should be provided

**Comment by billputer** [comment]
Agreed. I had a lot of trouble figuring this out when following the instructions for Autoscaling.
